src/main/java/com/dev/theater/dao/impl/ShoppingCartDaoImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.dev.theater.dao.impl;
+
+import com.dev.theater.dao.ShoppingCartDao;
+import com.dev.theater.exception.CrudException;
+import com.dev.theater.library.Dao;
+import com.dev.theater.model.ShoppingCart;
+import com.dev.theater.model.User;
+import com.dev.theater.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class ShoppingCartDaoImpl extends DaoImpl<ShoppingCart> implements ShoppingCartDao {
+    @Override
+    public ShoppingCart getByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("from ShoppingCart sc "
+                                        + "where sc.user = :user", ShoppingCart.class)
```

Comment:
The Better variant will be using fetch
```suggestion
            return session.createQuery("from ShoppingCart sc left join fetch sc.ticketList"
                            + " where sc.user = :user", ShoppingCart.class);
```